video_xblock/static/css/student-view.css

```diff
@@ -1,3 +1,16 @@
+.video-iframe-holder {
```

Comment:
Do we have a common stylesheet for LMS and Studio because we have duplicated code in two files?

Reply:
Removed from adjacent file